pkg/kubernetes/rbac/const.go

```diff
@@ -5,106 +5,108 @@
+	NodesResource                       = "nodes"
+	NodeStats                           = "nodes/stats"
+	PersistentVolumeClaimsResource      = "persistentvolumeclaims"
+	PersistentVolumesResource           = "persistentvolumes"
 	PodDisruptionBudgetsResource        = "poddisruptionbudgets"
+	PodSecurityPolicyResource           = "podsecuritypolicies"
+	PodsExecResource                    = "pods/exec"
+	PodsResource                        = "pods"
 	ReplicasetsResource                 = "replicasets"
-	DeploymentsResource                 = "deployments"
+	ReplicationControllersResource      = "replicationcontrollers"
+	ResourceQuotasResource              = "resourcequotas"
+	RoleBindingResource                 = "rolebindings"
+	RoleResource                        = "roles"
+	SecretsResource                     = "secrets"
+	ServiceAccountResource              = "serviceaccounts"
+	ServicesResource                    = "services"
 	StatefulsetsResource                = "statefulsets"
-	DaemonsetsResource                  = "daemonsets"
-	JobsResource                        = "jobs"
-	CronjobsResource                    = "cronjobs"
 	StorageClassesResource              = "storageclasses"
+	SubjectAccessReviewResource         = "subjectaccessreviews"
+	ValidatingConfigResource            = "validatingwebhookconfigurations"
 	VolumeAttachments                   = "volumeattachments"
-	ExtendedDaemonSetReplicaSetResource = "extendeddaemonsetreplicasets"
-	ServiceAccountResource              = "serviceaccounts"
-	NamespaceResource                   = "namespaces"
-	PodSecurityPolicyResource           = "podsecuritypolicies"
-	ClusterRoleBindingResource          = "clusterrolebindings"
-	RoleBindingResource                 = "rolebindings"
-	NetworkPolicyResource               = "networkpolicies"
-	IngressesResource                   = "ingresses"
 	VPAResource                         = "verticalpodautoscalers"
-	SubjectAccessReviewResource         = "subjectaccessreviews"
-	ClusterRoleResource                 = "clusterroles"
-	RoleResource                        = "roles"
+	WpaResource                         = "watermarkpodautoscalers"
 
 	// Non resource URLs
 
-	VersionURL     = "/version"
 	HealthzURL     = "/healthz"
-	MetricsURL     = "/metrics"
 	MetricsSLIsURL = "/metrics/slis"
+	MetricsURL     = "/metrics"
+	VersionURL     = "/version"
 
 	// Verbs
 
+	CreateVerb = "create"
+	DeleteVerb = "delete"
 	GetVerb    = "get"
 	ListVerb   = "list"
-	WatchVerb  = "watch"
-	UpdateVerb = "update"
 	PatchVerb  = "patch"
-	CreateVerb = "create"
-	DeleteVerb = "delete"
+	UpdateVerb = "update"
+	WatchVerb  = "watch"
 
-	// Rbac resource kinds
+	// RBAC resource kinds (singular)
 
-	ClusterRoleKind    = "ClusterRole"
-	RoleKind           = "Role"
-	ServiceAccountKind = "ServiceAccount"
+	ClusterRoleBindingKind = "ClusterRoleBinding"
+	ClusterRoleKind        = "ClusterRole"
+	RoleKind               = "Role"
```

Comment:
Should we consider also adding within the singular `RoleBindingKind = "RoleBinding"` as it was present previously in `internal/controller/datadogagent/const.go`, even though it's not used yet ?

Reply:
thanks for the review - since we're not using it i'll leave it out, but if ever do need it in the future it'll be clear where it goes